packages/files-ui/src/Components/Modules/FileBrowsers/hooks/useCreateOrEditSharedFolder.tsx

```diff
@@ -25,8 +25,7 @@ export const useCreateOrEditSharedFolder = () => {
       .then((bucket) => {
         setIsCreatingSharedFolder(false)
         return bucket
-      })
-      .catch((error) => {
+      }).catch((error) => {
```

Comment:
nit for consistency, I think it was fine as it was.
```suggestion
      })
      .catch((error) => {
```